tests/fsharp/Compiler/Language/BoolPartialActivePatternTests.fs

```diff
@@ -0,0 +1,105 @@
+﻿// Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information.
+
+namespace FSharp.Compiler.UnitTests
+
+open NUnit.Framework
+open FSharp.Compiler.Diagnostics
+open FSharp.Test
+
+[<TestFixture>]
+module BoolPartialActivePatternTests =
```

Comment:
Could you please port the new tests into the ComponentTests suite?
Especially when using just the 'typecheck' step (i.e. without full compilation and running), those are a lot faster which is better for regular CI runs.

Reply:
How should I do? I am not so familiar to the codebase

Reply:
I think it is basically moving this file to `tests/FSharp.Compiler.ComponentTests/Language` and make sure that this file is added to the `FSharp.Compiler.ComponentTests.fsproj`.

Reply:
tests moved